frontend/packages/dev-console/integration-tests/testData/YAMLs/create-job.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: example-job
+  namespace: aut-workloads-admin
+spec:
+  selector: {}
+  template:
+    metadata:
+      name: pi
+    spec:
+      containers:
+        - name: pi
+          image: perl
+          command:
+            - perl
+            - '-Mbignum=bpi'
+            - '-wle'
+            - print bpi(2000)
+      restartPolicy: Never
```

Comment:
Empty line is required at the end of every file, please make sure of it

Reply:
Updated